docs/development/contributing.rst

```diff
@@ -80,7 +80,7 @@ Technical Writers
 ~~~~~~~~~~~~~~~~~
 
 Technical writers and editors are invited to review the `documentation
-<https://docs.securedrop.org/>`__ and fix any mistakes in accordance with the
+<https://docs.securedrop.org/en/stable/>`__ and fix any mistakes in accordance with the
```

Comment:
In this case, and others where we currently link directly to a root of an RTD site like https://docs.securedrop.org/ and https://support-docs.securedrop.org/, I would prefer to keep doing so. That way, if we want to change the default version for any reason, we do not have to  update those URLs.

Reply:
This makes sense 👍 

I'll take a look at what settings are available in _linkcheck_ in case there is a way to avoid rendering the permanent redirect warnings in these cases. (I saw some ignored links there may be finer-grained options.)